cmd/iam_org_policy_reset.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/exoscale/cli/pkg/account"
+	"github.com/exoscale/cli/pkg/globalstate"
+	exoscale "github.com/exoscale/egoscale/v2"
+	exoapi "github.com/exoscale/egoscale/v2/api"
+)
+
+type iamOrgPolicyResetCmd struct {
+	cliCommandSettings `cli-cmd:"-"`
+
+	Force bool `cli-short:"f" cli-usage:"don't prompt for confirmation"`
+
+	_ bool `cli-cmd:"reset"`
+}
+
+func (c *iamOrgPolicyResetCmd) cmdAliases() []string { return nil }
+
+func (c *iamOrgPolicyResetCmd) cmdShort() string {
+	return "Reset Org policy to default"
+}
+
+func (c *iamOrgPolicyResetCmd) cmdLong() string {
+	return `This command resets IAM Organization Policy to default (allow all).
+This will remove any constraing that are set in Org Policy.`
```

Comment:
```suggestion
	return `This command resets the IAM Organization Policy to the default (allow all).
This will remove any constraints that were set in the Org Policy.`
```